src/peopleContext.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { Person } from './types';
+
+export const PeopleContex = React.createContext<Person[]>([]);
```

Comment:
There is a typo in the context name. It should be `PeopleContext` instead of `PeopleContex`.